### ISSUE TITLE
Remove border for iOS

### ISSUE THREAD

**Issue by amizz** [rhanb/nativescript-bottombar#55]
It seems that I can't remove the border on iOS. But it's working on Android.

<img width="1440" alt="screen shot 2018-01-29 at 12 15 58 am" src="https://user-images.githubusercontent.com/7664687/35484356-c64e4fc4-0489-11e8-9162-c8f6db342b27.png">

```
<BottomBar row="1" [items]="items" [hide]="hidden" [titleState]="titleState" (loaded)="tabLoaded($event)" (tabSelected)="tabSelected($event)" [inactiveColor]="inactiveColor" [accentColor]="accentColor" colored="true" borderTopWidth="0"></BottomBar>
```

**Comment by rhanb** [comment]
@amizz , thanks for this report. Really nice `BottomBar` by the way. Effectively, this feature is not supported in the `swift` library. Will ping you when it will be implemented 👍

**Comment by rhanb** [comment]
Should be fixed with the pre-release 4.0.0-rc1.1. Be careful, there is a lot of breaking changes.
